### PR TITLE
Update isct_antsSliceRegularizedRegistration to enable rotation

### PR DESCRIPTION
**Pull request by jcohenadad**
Fixes https://github.com/neuropoly/spinalcordtoolbox/issues/1178

Binaries for Debian/CentOS/OSX were updated with new build of ANTs (ae5a60bbca1db498a5ad3395b4218e341ba13357). Other binaries were not updated.

### CODE REVIEW

**Comment by jcohenadad** [comment]
Problem: antsSliceRegularizedRegistration is not behaving correctly. The output .csv (is regularized) does not correspond to the output warping field (is not regularized). An email was sent to B Avants.

File: install/copy_ants_binaries.py [hunk 1]
```diff
@@ -43,48 +38,41 @@ def usage():
...
 for file_ants in list_file_ants:
     # check if file exists
     if not os.path.exists(folder_ants+file_ants):
-        sct.printv('\nERROR: File '+folder_ants+file_ants+' does not exist.\n', 1, 'error')
+        print('\nERROR: File '+folder_ants+file_ants+' does not exist.\n')
     # copy data
-    sct.run('cp '+folder_ants+file_ants+' '+path_bin+prefix_sct+file_ants, 1)
-
-print "done!\n"
+    print('Copying: '+file_ants)
```
Comment (peristeri) [inline install/copy_ants_binaries.py]: For complete compatibility with all version of python, the future import is required `from __future__ import print_function` as the first import statement.

https://stackoverflow.com/questions/32032697/how-to-use-from-future-import-print-function

Comment (jcohenadad) [inline install/copy_ants_binaries.py]: agreed-- can you please modify accordingly? thanks

**Comment by peristeri** [comment]
I would like to merge this PR, but I would like more feedback from others. @benjamindeleener @poquirion can you review this PR too?

**Comment by jcohenadad** [comment]
please do NOT merge this PR yet. Still waiting for the bug correction by the ANTs team. Not relevant to have Ben/PO reviewing at this point.

**Comment by jcohenadad** [comment]
Brian Avants is not supporting this feature anymore. We close it for now.

File: scripts/sct_register_multimodal.py [hunk 2]
```diff
@@ -594,7 +594,7 @@ def register(src, dest, paramreg, param, i_step_str):
             scr_regStep = sct.add_suffix(src, '_regStep' + i_step_str)
             # estimate transfo
             cmd = ('isct_antsSliceRegularizedRegistration '
-                   '-t Translation[' + paramreg.steps[i_step_str].gradStep + '] '
+                   '-t Rigid[' + paramreg.steps[i_step_str].gradStep + '] '
```
Comment (joshuacwnewton) [inline scripts/sct_register_multimodal.py:597]: (Adding comment so I can more easily link to this change.)

**Pull request closed without merging**